sys/dev/drm/drmkpi/include/drm/drm_os_config.h

```diff
@@ -6,7 +6,9 @@
 #endif
 
 #define COMPAT_FREEBSD32 1
+#if 0 /* No CHERI support for 32 bit compat code. */
```

Comment:
We also don't have COMPAT_FREEBSD32... and this should probably be `!__has_feature(capabilities)`